tests/integration/test_base_repository_with_database.py

```diff
@@ -89,6 +148,32 @@ def test_raise_entity_not_found(pet_base_repository: PetBaseRepository, dog: Pet
             with pytest.raises(EntityNotFoundException):
                 pet_base_repository._update(entity=dog, name="new_name")  # type: ignore
 
+    class TestUpdateBatch:
+        """Tests for the _update_batch method"""
+
+        def test(self, pet_base_repository: PetBaseRepository, dog: Pet, cat: Pet, fish: Pet):
+            """Test to update an entity"""
+            new_name = "Fido II"
+            pet_base_repository._update_batch(entities=[dog, cat, fish], name=new_name)
+            updated_dog = pet_base_repository._get(entity_id=dog.id)
+            updated_cat = pet_base_repository._get(entity_id=cat.id)
+            updated_fish = pet_base_repository._get(entity_id=fish.id)
+
+            assert updated_dog.name == new_name
+            assert updated_dog.age == dog.age
+            assert updated_dog.type == dog.type
+            assert updated_dog.shelter_id == dog.shelter_id
+
+            assert updated_cat.name == new_name
+            assert updated_cat.age == cat.age
+            assert updated_cat.type == cat.type
+            assert updated_cat.shelter_id == cat.shelter_id
+
+            assert updated_fish.name == new_name
+            assert updated_fish.age == fish.age
+            assert updated_fish.type == fish.type
+            assert updated_fish.shelter_id == fish.shelter_id
+
```

Comment:
maybe also test what happens when you provide invalid attribute names (e.g. `pet_base_repository._update_batch(entities=[dog, cat, fish], rofl="copter")`) - I would expect that a proper exception is raised

Reply:
Valid point. Will add that